packages/components/src/components/hds/form/section/index.hbs

```diff
@@ -0,0 +1,17 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: MPL-2.0
+}}
+
+<div class={{this.classNames}} ...attributes>
+  {{yield
+    (hash
+      Header=(component "hds/form/section/header")
+      HeaderTitle=(component "hds/form/header/title" size="300")
+      HeaderDescription=(component "hds/form/header/description")
+      MultiFieldGroup=(component "hds/form/section/multi-field-group")
+      MultiFieldGroupItem=(component "hds/form/section/multi-field-group/item")
+      Section=(component "hds/form/section")
```

Comment:
[todo] maybe it was lost my comment, but I don't think you need to yield a section within a section

Reply:
Removing, thanks